src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -396,15 +396,15 @@ static LSP.Location ConvertTextSpanWithTextToLocation(TextSpan span, SourceText
         private static RequestDispatcher CreateRequestDispatcher(TestWorkspace workspace)
         {
             var factory = workspace.ExportProvider.GetExportedValue<RequestDispatcherFactory>();
-            return factory.CreateRequestDispatcher(ProtocolConstants.RoslynLspLanguages);
+            return factory.CreateRequestDispatcher(ProtocolConstants.RoslynLspLanguages, WellKnownLspServerKinds.AlwaysActiveLspServer);
```

Comment:
i'm a bit nervous about all tests just getting this particular server.  should tests indicate which servers to run against so we can test them all?

Reply:
So 99% of the time this is just used for telemetry purposes.   The functionality of all the servers is essentially the same and are instead gated based on the capabilities they provide.  They really shouldn't differ in functionality, it is only the special case of diagnostics that we need to know the server.  

I've added tests to verify the server based behavior for diagnostics, but otherwise left the default value to be this.